docs/background/disaster-recovery.md

```diff
@@ -0,0 +1,54 @@
+# Disaster recovery
+
+When you go on to create a new server in {{brand}}, you will probably
```

Comment:
Not sure what "go on to" means here. We might be able to drop it (as we might drop "probably")?

```suggestion
When you create a new server in {{brand}}, you will
```

Also, please make "create a new server" a link to the corresponding how-to guide.

Reply:
Done and done!